### ISSUE TITLE
Implement word prediction using Keras with tensorflow backend

### ISSUE THREAD

**Issue by z-zawhtet-a** [titipata/yelp_dataset_challenge#5]

**Comment by titipata** [comment]
Yeah, we should try sequence to sequence prediction using Keras. It's much simpler, great way to test our idea.

**Comment by titipata** [comment]
I will try: https://github.com/farizrahman4u/seq2seq for the character prediction

**Comment by daniel-acuna** [comment]
The tensorflow sequence to sequence model runs at 800 words per second on my 24 cores computer. If keras can do any better, then lets switch to it!

**Comment by titipata** [comment]
Keras won't do much better since we use tensorflow backend. Can you try running it on chunk of reviews?

**Comment by daniel-acuna** [comment]
can we change the backend? I think tensorflow is too slow
